### PR TITLE
Add VERAView.app

### PR DESCRIPTION
**Pull request by zbeekman**
VERAView is a GUI for visualization and engineering analyses of output
data from VERA (Virtual Environment for Reactor Applications).

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
      - complains about URL and Homepage miss-match. Comment added above URL block stating that I verified the download page & URL. If anything, the ornl.gov domain is more trustworthy than the github.com domain.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by zbeekman** [comment]
Can an Apache hosted directory act as an appcast? (i.e., new versions will show up there when posted...)? (See https://newton.ornl.gov/casl/)

File: Casks/veraview.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'veraview' do
+  version '2.4.3'
+  sha256 '57c7a10cb232ff69294f2aad731d84d3b6ced53ce6650bd4c48f1bb30bbbec80'
+
+  # https://newton.ornl.gov/casl was verified as official when first introduced to the cask
```
Comment (zbeekman) [inline Casks/veraview.rb]: Do I need to remove `https://` or something like that here to make the audit happy?

Comment (vitorgalvao) [reply Casks/veraview.rb]: Yes, we never use that part.

**Comment by vitorgalvao** [comment]
> Can an Apache hosted directory act as an appcast?

Yes.

**Review by vitorgalvao — APPROVED** [review]

**Comment by zbeekman** [comment]
Thanks @vitorgalvao! LGTM

**Pull request merged**